src/components/header/header.js

```diff
@@ -0,0 +1,134 @@
+import sheet from "./header.css" with { type: "css" };
+import discordIcon from "../../assets/discord.svg?type=raw";
+import githubIcon from "../../assets/github.svg?type=raw";
+import twitterIcon from "../../assets/twitter-logo.svg?type=raw";
+import mobileMenuIcon from "../../assets/tile.svg?type=raw";
+
+export default class Header extends HTMLElement {
+  constructor() {
+    super();
+    this.isMobileMenuActive = false;
+  }
+
+  connectedCallback() {
+    if (!this.shadowRoot) {
+      this.attachShadow({ mode: "open" });
+      this.shadowRoot.innerHTML = `
+        <header>
+          <div class="logo-bar">
+            <img class="greenwood-logo" src="/assets/greenwood-logo.svg" alt="Greenwood Logo">
+          </div>
+
+          <nav class="nav-bar">
+            <ul class="nav-bar-menu">
+              <li class="nav-bar-menu-item">
```

Comment:
Do we want to leverage any visited selector styles for our links?  Not sure if that's best practice or not, but they do show as purple for me (naturally)
![Screenshot 2024-05-17 at 3 56 28 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/c56e9159-493c-4f98-a281-8156234b89d5)

Not sure what best practices are here

cc: @aholtzman